week1/homework/src/server.js

```diff
@@ -5,13 +5,40 @@ const http = require('http');
 /* `createServer` MUST return an instance of `http.Server` otherwise the tests
  * will fail.
  */
+
+function handleRequest(response, state) {
+  response.writeHead(200, { 'Content-Type': 'application/json' });
+  response.write(JSON.stringify({ state }));
+}
+function handleError(response) {
+  response.writeHead(404, { 'Content-Type': 'application/json' });
+  response.write(JSON.stringify({ error: 'Nothing here!' }));
```

Comment:
The test is failing because the test tool expects the error response literally `Not found`, so please change it to make test pass.

Reply:
Thanks for your feedback! It's done 👍 :)